build_library/generate_kernel_hash.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/python
+
+import hashlib
+import json
+import os
+import sys
+
+rootdir=sys.argv[1]
+version=sys.argv[2]
+
+path = rootdir + "/usr/boot/vmlinuz"
```

Comment:
Can we point this at the ESP copy of the kernel (/boot) so we don't forget about this later when enabling verity?